libsrc/target/sms/set_vdp_reg.asm

```diff
@@ -1,23 +0,0 @@
-	SECTION	code_clib	
```

Comment:
This routine is not used. Instead the one from .../video/tms9918/gen_set_vdp_reg.asm is used.